dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -273,9 +405,11 @@ public void addTasksTo(List<? super Task<?>> out, @Nonnull ConnectorArguments ar
         ZonedIntervalIterable intervals = ZonedIntervalIterable.forConnectorArguments(arguments);
         LOG.info("Exporting query log for " + intervals);
         SharedState state = new SharedState();
+        boolean isAssessment = arguments.isAssessment();
         for (ZonedInterval interval : intervals) {
             String file = ZIP_ENTRY_PREFIX + DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(interval.getStartUTC()) + ".csv";
-            out.add(new TeradataLogsJdbcTask(file, state, logTable, queryTable, condition, interval).withHeaderClass(Header.class));
+            out.add(new TeradataLogsJdbcTask(file, state, logTable, queryTable, condition, interval,
+                    isAssessment).withHeaderClass(isAssessment ? HeaderForAssessment.class : Header.class));
```

Comment:
I think this should probably better be a subclass (`TeradataAssessmentLogsJdbcTask`) rather than fiddling with booleans and changing arrays.